integration/spark/app/src/test/java/io/openlineage/spark/agent/SparkContainerIntegrationTest.java

```diff
@@ -269,10 +269,7 @@ void testDropTable() {
   void testTruncateTable() {
     SparkContainerUtils.runPysparkContainerWithDefaultConf(
         network, openLineageClientMockContainer, "testTruncateTable", "spark_truncate_table.py");
-    verifyEvents(
-        mockServerClient,
-        "pysparkTruncateTableStartEvent.json",
-        "pysparkTruncateTableCompleteEvent.json");
+    verifyEvents(mockServerClient, "pysparkTruncateTableCompleteEvent.json");
```

Comment:
we don't expect the start event anymore?

Reply:
Expecting this on complete is enough as events are cumulative.